### ISSUE TITLE
Do we document BJ functions as well?

### ISSUE THREAD

**Issue by Cokemonkey11** [lep/jassdoc#1]
I think it could be interesting to know how certain BJ functions work, for example cinematic/UI functions, and melee initialization functions.

**Comment by lep** [comment]
Yes. I think it would be worthwile to extract blizzards hints out of `triggerstrings.txt`.

**Comment by Cokemonkey11** [comment]
See #24

What is `triggerstrings.txt` ?

**Comment by Luashine** [comment]
I'm not sure if the file changed its name or I got a different one: CASC of 1.32 `war3.w3mod\ui\triggerdata.txt` - contains GUI Trigger data needed for code generation, descriptions and the parameters. Strings are localized so they're stored elsewhere like `WESTRING_UNITORDERNOTARG_STOP`.
